solution/ui/regulations/utilities/api.js

```diff
@@ -286,8 +286,20 @@ const getPartTOC = async (title, part) =>
 
 const getCategories = async () => httpApiGet("resources/categories");
 
```

Comment:
Just like we'd like all components to be in a single location, it'd be great if all the utility methods, api methods, and filter methods in a single place instead of having duplicate files in each project.

For now, there is still an `api.js` file in the `eregs-component-lib` project.  The remaining methods there still need to be refactored to be isomorphic and work in either project.  A story will be or has been filed to do this.